x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/math/AutoBucketTests.java

```diff
@@ -80,12 +80,13 @@ public static Iterable<Object[]> parameters() {
         })));
     }
 
+    @SuppressWarnings("checkstyle:LineLength")
```

Comment:
We should just run the formatter I think.

Reply:
done

Reply:
Could you remove this suppression as well? That'll make sure it never grows too long accidentally.